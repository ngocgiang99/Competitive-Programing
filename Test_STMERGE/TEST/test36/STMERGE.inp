2
3 5
231903697 708712098 916960965 569587209 357630189 
585616818 245297077 260763388 43220495 515554313 
403798254 560428095 40056780 935065290 49559333 
2 4
509845339 197590351 250795786 373535917 
318554464 561652897 943089627 568567355 
