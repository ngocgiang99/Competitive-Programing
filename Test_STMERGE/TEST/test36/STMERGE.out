324683525
318554464
